Library/ViewModels/DiscoveryViewModelTests.swift

```diff
@@ -43,20 +43,6 @@ internal final class DiscoveryViewModelTests: TestCase {
     self.configureDataSource.assertValueCount(1, "Data source configures after view loads.")
   }
 
-  func testConfigureDataSourceOptimizelyConfiguration() {
```

Comment:
Ah good catch. Though the it should be a XCTAsserFalse assertion since we're no longer `activating` an experiment. Activating is what would set `activatePathCalled` to true.

[done](https://github.com/kickstarter/ios-oss/pull/1812/commits/771b6b53d9ed65a8b190a67e6801b5e1d85a3398)